test_fitness.py

```diff
@@ -92,33 +93,34 @@ def write_chi(chi2):
 
 # Define target gain pattern based on design
 if (g.design == "ARA"):
-    target = [[0.968807, 14.1263, -0.053024, 0.8256],
-              [4.83102, 10.4062, 0.027465, -0.396757]]
+    target = ([[0.968807, 14.1263, -0.053024, 0.8256],
+              [4.83102, 10.4062, 0.027465, -0.396757]])
```

Comment:
```suggestion
    target = [[0.968807, 14.1263, -0.053024, 0.8256],
              [4.83102, 10.4062, 0.027465, -0.396757]]
```
No need for extra `()` when you already have `[]` or `{}`.